<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>181</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>531</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolButton" name="lasttoolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/pre.png</normaloff>:/img/pre.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/pause.png</normaloff>:/img/pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="nexttoolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/next.png</normaloff>:/img/next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="sjlabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/file.png</normaloff>:/img/file.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/loop.png</normaloff>:/img/loop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/img/volume.png</normaloff>:/img/volume.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
